### DISCUSSION TITLE
🐛 Bug Report: GitHub Education's "Sign in" Button Leads to a Dead End

### DISCUSSION THREAD

**Discussion by luke-beep** [community/community#75299] [GitHub Education] [answered]
## 🔍 Issue Overview:

When attempting to sign in to GitHub Education, clicking the "Sign in" button results in a page that either does not load or leads to a dead end, preventing further access.

Steps to Reproduce
--------------------------

Navigate to GitHub Education's [homepage](https://education.github.com/).
Click on the "Sign in" button.
Observe the resulting behavior or page.

Expected Behavior
----------------------

✅ What should happen:

Clicking the "Sign in" button should redirect users to a login page where they can enter their credentials.

Actual Behavior
----------------------

❌ What actually happens:

The "Sign in" button redirects to a non-functional page or leads to a dead end, impeding the login process.

Environment
------------------

Browser: Brave 1.59.124
OS: Windows NT 10.0 (Windows 11 23H2 @ Build 25992.1000)
Device: Desktop

Labels
------------

bug help wanted documentation good first issue

**Comment by MohamedNourDerbeli** [comment]
It appears that you're experiencing an issue with the GitHub Education sign-in process using the Brave browser on Windows 11. This could be due to various reasons, such as browser settings, extensions, or potential compatibility issues. Here are some suggestions to help you troubleshoot and resolve the issue:

1. **Try a Different Browser:**
   - First, attempt to sign in using a different browser to see if the issue persists. This can help determine whether the problem is specific to the Brave browser.

2. **Disable Browser Extensions:**
   - Browser extensions can sometimes interfere with website functionality. Try disabling all browser extensions, especially those related to privacy or security, and see if you can sign in.

3. **Clear Browser Cache and Cookies:**
   - Cached data or cookies may cause issues. Clear your browser's cache and cookies, and then try to sign in again.

4. **Browser Compatibility Mode:**
   - Ensure that your browser is not running in compatibility mode. GitHub is designed to work with modern browsers, and running in compatibility mode might cause unexpected issues.

5. **Update Browser:**
   - Make sure that you are using the latest version of the Brave browser. If not, update the browser to the latest version, as newer releases often include bug fixes and improvements.

6. **Incognito/Private Browsing Mode:**
   - Try signing in using the browser's incognito or private browsing mode. This can help identify if the issue is related to browser settings or extensions.

7. **JavaScript Enabled:**
   - Ensure that JavaScript is enabled in your browser settings, as many modern websites, including GitHub, rely on JavaScript for interactive features.

8. **Network Issues:**
   - Check if there are any network issues that might be affecting your connection to GitHub. Ensure that your internet connection is stable.

9. **Contact GitHub Support:**
   - If the issue persists after trying the above steps, consider reaching out to GitHub support for assistance. They can provide specific guidance and may be aware of any known issues.

If the problem continues, it could be a specific issue with the GitHub Education platform or a bug that needs to be reported. In that case, consider submitting a bug report or reaching out to GitHub Education support for further assistance.

**Reply by luke-beep** [reply]
I've already tried on my school laptop which is running Windows 10, and my phone which is an iPhone 12. Same results.

**Comment by ettaboyle** [comment]
@luke-beep I have been unable to reproduce this in an incognito browser, but have shared a link to this discussion in case its just something I am missing.

**Reply by ettaboyle** [reply]
@luke-beep Can you try signing into [github.com](http://github.com/) and then go to https://education.github.com/globalcampus/student and see what happens 🤞🏻

**Reply by luke-beep** [reply]
It redirects me to the official GitHub education page, but when I'm signed out it gives me an error every single time on all of my devices.

**Reply by ettaboyle** [reply]
Can you please add screenshots or a screen recording we can share with the engineering team? As we have been unable to reproduce this issue we're unsure how to move forward to help you.

**Reply by luke-beep** [reply, marked as answer]
Okay, now it has been fixed for some weird reason.
